arrays.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var namesArray = ["kane","dan","dom","andrew","liam","laurie","ian","katherine","sohaib"];
+
+console.log(namesArray[namesArray.length-1]);
+
+console.log(namesArray[0]);
+
+var namesString = namesArray.join();
+console.log(namesString);
+
+var ourNames = "kane,dan,dom,andrew,liam,laurie,ian,katherine,sohaib";
+
+var listONames=ourNames.split(',');
+
+console.log(listONames);
+
+var brandnewArray = namesArray.concat(['keith','steve','darren']);
+
+console.log(brandnewArray);
+
+var randomNumber = [(Math.random()*100),(Math.random()*100),(Math.random()*100),(Math.random()*100)];
+
+for(var i =0;i<randomNumber.length;i++){
+    if(randomNumber[i]%3===0){
```

Comment:
Missed this - convert this into a function that takes an array, and returns an array which has items form the first which divide by 3.

The idea is that functions are re-useable, so you make them do the bare minimum amount of work to get the job done. In this case, outputting wouldn't be part of the function's job, you output from the test code.